src/Components/Dialogs/dialogs.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 
 import style from './dialogs.module.css';
+import I18Y from '../../core/constants';
+import GetItem from './GetItems/getItem';
 
 const Dialogs = () => (
-  <div className={style.profile}>
-    Dialogs ve been here
+  <div className={style.dialogs}>
+    <div className={style.dialogsItems}>
+      <GetItem nambers={I18Y.name} />
```

Comment:
There's a typo in "nambers" - it should be "numbers"

Reply:
Same as above